### ISSUE TITLE
collision between groovy LICENSE openjdk LICENSE

### ISSUE THREAD

**Issue by pmahoney-raise** [NixOS/nixpkgs#19872]
## Issue description

There is a conflict installing groovy and openjdk in the same nix profile. The LICENSE file exists at the toplevel; it should probably be located elsewhere for both packages.
### Steps to reproduce

```
$ nix-env -iA nixpkgs.groovy nixpkgs.openjdk
...
collision between ‘/nix/store/mjd2gsnx14q6n05f8fwjk0ry6kms0m40-groovy-2.4.6/LICENSE’ and ‘/nix/store/g26vd7kl13vsfpghs9pks4avrwl8drng-openjdk-7u60b30/LICENSE’; use ‘nix-env --set-flag priority NUMBER PKGNAME’ to change the priority of one of the conflicting packages
```
## Technical details
- System: OS X 10.10.5 Yosemite
- Nix version: (run `nix-env --version`) nix-env (Nix) 1.11.4
- Nixpkgs version: (run `nix-instantiate --eval '<nixpkgs>' -A lib.nixpkgsVersion`) 17.03pre94294.a057dfc

**Comment by Mic92** [comment]
I think you could make a pull request, where you delete LICENSE in both package, within `postInstall`.

**Comment by NeQuissimus** [comment]
May be related to what I am seeing with `bind` in #19761

**Comment by sternenseemann** [comment]
Wouldn't it be more sensible to move it to, say, `${out}/share/<pkgname>/LICENSE` in `postInstall`?

**Comment by vcunat** [comment]
Yes, this stuff should not be installed in top level.

**Comment by sternenseemann** [comment]
Okay, I will fix that.

**Comment by NeQuissimus** [comment]
I think only people with commit access can be assigned. But it's yours :)

**Comment by calvertvl** [comment]
@pSub: It looks like this can be closed, unless you want to cherry pick to 16.09
